src/ralph/conf.py

```diff
@@ -133,6 +133,36 @@ class Config:  # pylint: disable=missing-class-docstring # noqa: D106
     timeout: float
 
 
+class AuthBackend(Enum):
```

Comment:
```suggestion
class AuthBackend(StrEnum):
```

Reply:
made the change with `class AuthBackend(str, Enum):`